control/runtimes/bridge-hub-polkadot/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+#[subxt::subxt(
+    runtime_metadata_path = "bridge-hub-metadata.bin",
+    derive_for_all_types = "Clone",
+    substitute_type(
+        path = "snowbridge_beacon_primitives::updates::CheckpointUpdate",
+        with = "::subxt::utils::Static<::snowbridge_beacon_primitives::updates::CheckpointUpdate<512>>",
```

Comment:
Interesting, is that because of the generic parameters `COMMITTEE_SIZE` which `subxt` can't infer?